### PR TITLE
Switch to swtpm for running TPM tests

### PR DESCRIPTION
**Pull request by shjala**
This PR replaces current TPM testing script, which is a bit extra (spawning a whole ubuntu VM to run TPM required test), with just using swtpm. It's clean and fast.

### CODE REVIEW

**Comment by shjala** [comment]
as suggested by @christoph-zededa , I will try to make it part of the "Test" job instead.

**Review by rucoder — APPROVED** [review]
LGTM

File: tests/tpm/prep-and-test.sh [hunk 1]
```diff
@@ -0,0 +1,107 @@
+
```
Comment (eriknordmark) [inline tests/tpm/prep-and-test.sh:1]: First line is blank hence the bin/bash will be ignored I think.

**Review by eriknordmark — APPROVED** [review]
LGTM

**Pull request merged**